### ISSUE TITLE
other file extensions

### ISSUE THREAD

**Issue by pwright** [haydenull/logseq-plugin-open-in-external-app#1]
Works great for excalidraw (thanks!)but I'm not sure how to apply it to md files, trying with:

```
{
    "menus": [
      {
        "menuName": "Excalidraw",
        "pathRegExp": "\\[\\[([\\d\\D]+)]]",
        "extensionName": "excalidraw",
        "apps": [
          {
            "title": "vscode",
            "openSchema": "vscode://file/{path}"
          }
        ]
      },
      {
        "menuName": "Markdown",
        "pathRegExp": "\\[\\[([\\d\\D]+)]]",
        "extensionName": "md",
        "apps": [
          {
            "title": "vscode",
            "openSchema": "vscode://file/{path}"
          }
        ]
      }
    ]
  }

```

**Comment by haydenull** [comment]
Can you write an example of your block？

Your current spelling can match the following format：
`[[pages/yourfile.md]]`

**Comment by pwright** [comment]
right, that kinda works, but I was hoping to be able to use vscode to open an arbitrary md file, or even something like `[[yourfile]]`

**Comment by haydenull** [comment]
This may not be possible. The plugin locates the file by splicing the path of the graph with the handwritten path in the block.

In addition, logseq contains the function of opening md files.

<img width="502" alt="image" src="https://user-images.githubusercontent.com/17695989/160741721-316e8b3f-8575-40bb-98ef-4ae590a14f9c.png">

**Comment by 0w0miki** [comment]
> right, that kinda works, but I was hoping to be able to use vscode to open an arbitrary md file, or even something like `[[yourfile]]`

If you want to open an arbitrary md file on you computer, you could try add enough `../` in the block.

For example, suppose you logseq notes located at `/home/user/notes/`. You could open a file at `/home/user/files/test.md` with `[[../files/test.md]]`. This works for me.

But I doubt if it is reasonable to link "xxx.md"

---
For the thoughts directly opening `[[yourfile]]`, I don't think it is a good idea.

Logseq now supports hierarchy page like `[[A/B]]`, which is saved as `A.B.md`. This brings big problems to the feature you want.
